backend/authentication/user.py

```diff
@@ -31,11 +30,11 @@ def is_authenticated(self) -> bool:
     @property
     def display_name(self) -> str:
         """Return username and discriminator as display name."""
-        return f"{self.payload['username']}#{self.payload['discriminator']}"
+        return f"{self.payload["username"]}#{self.payload["discriminator"]}"
```

Comment:
Absolutely brilliant. Death to the single quotes. They can no longer take refuge in f-strings.